app/models/email_address.rb

```diff
@@ -1,7 +1,52 @@
 class EmailAddress < ContactInfo
   include EmailAddressValidations
 
+  WHITELIST = [
+    # popular email providers globally
+    'gmail.com', 'outlook.com', 'yahoo.com', 'icloud.com', 'hotmail.com', 'aol.com',
+    # popular ones currently in our DB - as of Feb 2019
+    'rice.edu',
+    'openstax.org',
```

Comment:
@jpslav what is the purpose of checking it frequently - just to make sure our records are working? Or to verify the DNS check is working?

Reply:
@jpslav Removed rice.edu and openstax.org per your suggestion.
@mwharrison I'm going to guess that it's in order to verify that the DNS check is working. If it ever came back as invalid for those two domains, then we have a serious problem—it would mean that a bug is preventing new users from signing up. Hm... which btw, is something I would REALLY like to prevent and I'm open to suggestions. I was thinking about adding a time-out as a fail-safe; should I do it?

Reply:
Yes, the idea for checking it frequently is just for our users (who mostly use @rice.edu and @openstax.org) to put mileage on it over time.